examples/md-flexible/src/Simulation.cpp

```diff
@@ -644,7 +643,7 @@ void Simulation::logMeasurements() {
     std::cout << timerToString("    Thermostat                    ", thermostat, maximumNumberOfDigits, simulate);
     std::cout << timerToString("    Vtk                           ", vtk, maximumNumberOfDigits, simulate);
     std::cout << timerToString("    LoadBalancing                 ", loadBalancing, maximumNumberOfDigits, simulate);
-    std::cout << timerToString("One iteration                     ", simulate / static_cast<long>(_iteration),
+    std::cout << timerToString("    One iteration                 ", simulate / static_cast<long>(_iteration),
```

Comment:
```suggestion
    std::cout << timerToString("  One iteration                 ", simulate / static_cast<long>(_iteration),
```
You are right that this doesn't make sense at the "root" level, but I would suggest it should be flush with simulate as it is not a "component" of simulate.